src/vehicle.h

```diff
@@ -32,6 +32,7 @@
 #include "item_location.h"
 #include "item_stack.h"
 #include "line.h"
+#include "npc.h"
```

Comment:
I think you can avoid including `npc.h` and keep the forward declaration of `class npc` by declaring `~turrent_cpu()` in `turret_cpu` and defining it with `turret_cpu::~turret_cpu() = default;` in vehicle.cpp.

Reply:
It refuses to compile :P

Reply:
Thanks for approving, but I cannot figure out how to make it compile again without including `npc.h`. Should I just revert the change and include `npc.h`?

Reply:
OK I figured it out.